### PR TITLE
Add activity item for when thread privacy is changed

### PR DESCRIPTION
**Pull request by HSalmon**
[Trello card](https://trello.com/c/LhOfstmI/707-activity-item-for-changing-thread-privacy)

screen shot 2016-02-24 at 5 50 37 pm

### CODE REVIEW

**Comment by HSalmon** [comment]
There are probably a couple of things in here that could be more concise - would love some eyes when someone's got a mo'!

**Comment by robguthrie** [comment]
Nice! I don't think you need to say what the privacy was. There is only
ever one other option. Can I suggest "Actor changed the thread privacy to
public"

File: angular/core/components/thread_page/activity_card/discussion_edited_item_controller.coffee [hunk 1]
```diff
@@ -7,6 +7,17 @@ angular.module('loomioApp').controller 'DiscussionEditedItemController', ($scope
       version.changes.title[1]
     else
       ''
+  $scope.onlyPrivacyEdited = ->
+    version.attributeEdited('private') and
```
Comment (gdpelican) [inline angular/core/components/thread_page/activity_card/discussion_edited_item_controller.coffee]: Isn't this just `_.keys(version.changes) == ['private']`?

**Comment by HSalmon** [comment]
screen shot 2016-02-24 at 7 42 34 pm

**Comment by robguthrie** [comment]
sorry to be picky but can you drop the colon?

**Pull request merged**